src/Component/Field.js

```diff
@@ -0,0 +1,275 @@
+                placeholder={field.attributes.placeholder}
+                maxLength={field.attributes.maxLength}
+                minLength={field.attributes.minLength}
+                onChange={(e) => setText(e.target.value)}
+                size={field.attributes.size}
+                list={field.attributes.list}
+                onChange={(e) => handleOnchange(e,field)}
+                onBlur={(e) => handleBulr(e,field)}
+                required={field.validation.required}
+            />)
+          break;
+          case 'time':
+            element = (<input
+              type={field.fieldConfig.type}
+              name={field.attributes.name}
+              placeholder={field.attributes.placeholder}
+              maxLength={field.attributes.maxLength}
+              minLength={field.attributes.minLength}
+              onChange={(e) => setText(e.target.value)}
+              size={field.attributes.size}
+              list={field.attributes.list}
+              onChange={(e) => handleOnchange(e,field)}
+              onBlur={(e) => handleBulr(e,field)}
+              required={field.validation.required}
+          />)
+        break;
+            case 'radio': 
+                 element = (<><label>{field.attributes.value}</label><input
+                      type={field.fieldConfig.type}
+                      name={field.attributes.name}
+                      placeholder={field.attributes.placeholder} 
+                      onChange={(e) => setText(e.target.value)}
+                      value={field.attributes.value}     
+                      onChange={(e) => handleOnchange(e,field)}
+                      onBlur={(e) => handleBulr(e,field)}
+                      required={field.validation.required}
+                    /></>)
+        }
+        return element
+      }
+      
+
+    const createField = () => {
+        fieldData.map(field => {
+              switch(field.fieldName) {
+                case 'input':
+                    element =  checkInputTypeField(field);
+                    allFields.push(element)
+                   break;
+                 case 'select':
+                     element = createSelectField(field);
+                     allFields.push(element)
+                   break;
+                 case "button": 
+                      element = createButton(field);
+                      allFields.push(element);
+                    break;
+                  case 'textarea':
+                      element = createTextArea(field);
+                      allFields.push(element);
+                    break;
+                  case 'label': 
+                       element = createLabel(field);
+                       allFields.push(element);
+                    break;
+               }  
+        })
+        return allFields;
+    }
+
+
+    return (
+        <div >
+           <div className="container">
+            <div className="form"> 
+              {createField().map((field,i) => {
```

Comment:
Can we createField() once and use the output result?
Instead of getting it on every render

Reply:
yes right